microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
+__version__ = "1.01b"
+__last_update__ = "12/19/2016"
+
+
+import click
+import sys
+from skbio import io, Sequence
+
+
+def extract_sequences(infile, seqidx=0):
+    """ Extract sequence(s) from a multi-sequence FASTA file
+
+    Parameters
+    ----------
+    infile : str
+        file path to input multi-sequence FASTA file
+    seqidx : int (optional)
+        n-th sequence of the input file to extract
+        (default: 0 for all sequences)
+
+    Returns
+    -------
+    list of skbio Sequence
+    """
+    seqs = []
+    iseq = 0  # current sequence index
+    for seq in io.read(infile, format='fasta'):
+        iseq += 1
+        if 0 < seqidx != iseq:
```

Comment:
wow, never saw a condition like that. Does it translate to `if (0 < seqidx) and (seqidx != iseq):` is that a flake8 compatible style?